pages/content/ch3/sets.mdx

```diff
@@ -0,0 +1,80 @@
+import { Callout, Tabs} from 'nextra/components'
+import Problem from '@/components/problem'
+
+# The most basic object
+
+In mathematics a set constitutes the most basic object. We define
+a mathematical object by the properties of the operations we can perform on it.
+
+In the case of sets, we only have a single operation: the membership predicate taking two
+sets and mapping them to a truth value. In infix notation:
+$$
+x \in A \\
+x \notin A \iff \neg (x \in A)
+$$
+
+We can write a set by enumerating its elements:
+$$
+\{ A, B, C \}
+$$
+
+<Callout type="warning" emoji="💡">
+  When enumerating sets, the order of elements does not matter. Elements
+  can be repeated without an effect:
+    $$ \{ A, B, C \} = \{ A, C, B \} = \{ A, B, C, A \} $$
+</Callout>
+
+The membership predicate has the following properties (simplified version of the *ZF axioms*).
```

Comment:
Diese Axiome sind theoretisch nicht das was in der Vorlesungs erzaehlt wird, das ist also fuer die meisten irrelevant und evtl. verwirrend

Reply:
und sind das die "standard" axiome? auf wikipedia ist 1-3/1-4 anders ... https://en.wikipedia.org/wiki/Zermelo%E2%80%93Fraenkel_set_theory

(und was ist mit axiom of choice?)

Reply:
Tatsächlich sind es nicht die orginalen ZF axiome, sondern die definitionen aus dem Skript. Diese entsprechen jedoch ungefähr den Axiomen. 
Es fehlen auch 3 Axiome.